utils/gnbOperations.tsx

```diff
@@ -15,12 +14,44 @@ export async function getGnbList(token: string): Promise<GnbItem[]> {
     if (!response.ok) {
       throw new WebconsoleApiError(response.status, gnbList.error);
     }
-    return gnbList.map((gnb: GnbItem) => ({
-      ...gnb,
-      tac: Number(gnb.tac),
-    }));
+    return gnbList;
   } catch (error) {
     console.error(`Error retrieving gNB list ${error}`);
     throw error;
   }
 };
+
+interface EditGnbArgs {
+  name: string;
+  tac: number;
```

Comment:
```suggestion
  tac: string;
```
Webconsole endpoint expects to get TAC value as string.